alembic-autogenerate.py

```diff
@@ -42,7 +40,7 @@
 
 # Loop over each of the instruments
 pattern = os.environ["SDM_SCHEMAS_DIR"] + "/yml/cdb_*.yaml"
-instruments = [re.search(r"cdb_(.+)\.yaml$", file).group(1) for file in glob.glob(pattern)]
+instruments = ["latiss", "lsstcomcam", "lsstcomcamsim"]
```

Comment:
> We will need to include the glob again after I add the startracker tables

Or you can just add the new schemas to the instrument list. It's a small enough set of files that it can just be maintained by hand in this script, I think.

Reply:
I was at first agreeing with you, then thinking through what it would mean to add new instruments. (will we continue to add any more new instruments ever??) 

When one adds a new table, that instrument would have to be manually added here; 
If they try to add a new table to the schema, but don't know to edit this list, they won't get an error.
(wouldn't receive the 'no section in alembic.ini file' error)

I think I would prefer it to give an error when one attempts to add a cdb_*.yml rather than a success on an empty or no migration created. maybe accidentally skipping the 'need to test in TTS, etc' step. 

opinions welcome.

Reply:
> I was at first agreeing with you, then thinking through what it would mean to add new instruments. (will we continue to add any more new instruments ever??) 
> 
> 
> 
> When one adds a new table, that instrument would have to be manually added here; 
> 
> If they try to add a new table to the schema, but don't know to edit this list, they won't get an error.
> 
> (wouldn't receive the 'no section in alembic.ini file' error)
> 
> 
> 
> I think I would prefer it to give an error when one attempts to add a cdb_*.yml rather than a success on an empty or no migration created. maybe accidentally skipping the 'need to test in TTS, etc' step. 
> 
> 
> 
> opinions welcome.

I don't have any strong opinion but adding a new schema is uncommon enough that hard-coding the list doesn't seem so bad to me. If you would like the glob added back, I am okay with it. We just need to make sure that every cdb schema in sdm_schemas has a migration set or it will fail.